index.html

```diff
@@ -1 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <header id="header">
+        Product landing page
+         <img id="header-img" src="dat8zlJpGUVKLUoyScZRa/9k=" alt="hello">
+         <nav id="nav-bar">
```

Comment:
though the page works well, please use classes instead of ids as classes can be used on other sections as well to reduce redundancy